### ISSUE TITLE
cannot reach luci through zerotier

### ISSUE THREAD

**Issue by Suxsem** [mwarning/zerotier-openwrt#84]
There is something REALLY strange in reaching luci from zerotier.

try to do so: ssh into your openwrt device from the zerotier network.
Then send " /www/cgi-bin/luci". The connection drops after a few lines:

```
BusyBox v1.31.1 () built-in shell (ash)

  _______                     ________        __
 |       |.-----.-----.-----.|  |  |  |.----.|  |_
 |   -   ||  _  |  -__|     ||  |  |  ||   _||   _|
 |_______||   __|_____|__|__||________||__|  |____|
          |__| W I R E L E S S   F R E E D O M
 -----------------------------------------------------
 OpenWrt SNAPSHOT, r15668-d33cd383ed
 -----------------------------------------------------
root@Wind3HUB:~# /www/cgi-bin/luci
Status: 403 Forbidden
X-LuCI-Login-Required: yes
Content-Type: text/html
Cache-Control: no-cache
Expires: 0
X-Frame-Options: SAMEORIGIN
X-XSS-Protection: 1; mode=block
X-Content-Type-Options: nosniff

<!DOCTYPE html>
<html lang="en">
        <head>
                <meta charset="utf-8">


[connection drop]
```

how is that possible?

**Comment by goshil** [comment]
Hi. Not sure, if it could help, but try it:

_If you experienced issues with zerotier ssh and webui. Such as putty hangs with outputting 10+ lines.
Try to change MTU from default 2800 to this or smaller value `ifconfig ztzlgf7vul mtu 1388`_

Source: https://www.snbforums.com/threads/a-guide-about-installing-zerotier-on-asus-ac68u-router.42648

**Comment by Suxsem** [comment]
it seems to be solved on openwrt 21.02rc1 and the latest version of zerotier (1.6.5-1).
I will keep monitoring

**Comment by ogarcia** [comment]
Close because solved
